src/components/ModifierPlayground/PlaygroundChart/index.tsx

```diff
@@ -118,9 +79,14 @@ const PlaygroundChart: React.FC<{
   };
 
   const drawEquation = (ctx: CanvasRenderingContext2D) => {
-    const { minX, minY, maxX, maxY, translate } = prepareChartValues(
-      ctx.canvas
-    );
+    const canvas = ctx.canvas;
+    const { minX, minY, maxX, maxY, translate } = useChartValues({
+      canvas,
+      easingFunctionName,
+      easingNestType,
+      enlargeCanvasSpace,
+      isMobile,
+    });
```

Comment:
This `useChartValues` hook takes way too many parameters to be useful in any other place in the code. Extracting just for the sake of creating a new hook isn't very useful in this setting.